src/ResourceHeader.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const ResourceHeader = ({ label }) => {
+  return <span>{label}</span>
```

Comment:
should be ok to do `<>{label}</>` and not add an extra span

Reply:
sure.